domain-management/src/main/java/org/jboss/as/domain/management/logging/DomainManagementLogger.java

```diff
@@ -1127,6 +1127,15 @@
     String filePermissionsProblemsFound(String file);
 
     /**
+     * A message indicating an unsupported resource in the model during marshalling.
+     *
+     * @param name the name of the resource.
+     * @return The exception for the error.
+     */
+    @Message(id = 108, value = "Unsupported resource '%s'")
+    IllegalStateException unsupportedResource(String name);
```

Comment:
It's in ManagementXml_4.java - for some reason GitHub is not expanding that one by default.